content/abstracts.rst

```diff
@@ -108,13 +108,21 @@ The PyCon UK panel will probably include:
    of Wolverhampton. Loves to talk about distributed Python and
    concurrency. Is Chair of the science track at PyCon UK.
 
- * Van Lindberg is a smiley chap and Vice President of Intellectual
+ * Van Lindberg: a smiley chap and Vice President of Intellectual
    Property at Rackspace and chairman of the PSF.
 
- * Larry Hastings is one of the main CPython core contributors and the
+ * Larry Hastings: one of the main CPython core contributors and the
    Release Manager for Python releases.
 
- * A currently unselected Djangogirls organiser
+ * Nicholas Tollervey: freelance Python developer, classically
+   trained musician, philosophy graduate, teacher, writer, organiser of
+   the education track, founder of the London Python Code Dojo and PSF
+   ambassador to the BBC micro:bit project. He enjoys spending his
+   copious free time playing the tuba or hacking on distributed hash
+   tables.
+
+ * Ola Sendecka: Python and Django enthusiast. Django Girls
```

Comment:
@zeth, has Ola replied?  Don't think we should add people to this list who haven't said yes yet.